win32/gui.py

```diff
@@ -111,7 +115,7 @@
     }
 
 
-log = logging.getLogger(__name__)
+log = logging.getLogger('gui')
```

Comment:
An experiment to try to actually get the logs somewhere, but it didn't succeed. I'll revert it.